spanner/api/Spanner/Program.cs

```diff
@@ -1860,7 +1996,31 @@ await connection.BeginTransactionAsync())
                 Console.WriteLine("Inserted data.");
             }
         }
-        // [END spanner_insert_data]
+
+        public static async Task<object> DropDeleteSampleTables(
+            string projectId, string instanceId, string databaseId)
+        {
+            string connectionString =
+                $"Data Source=projects/{projectId}/instances/{instanceId}"
+                + $"/databases/{databaseId}";
+            // Make the request.
+            using (var connection = new SpannerConnection(connectionString))
+            {
+                foreach (string table in new[] { "UpcomingAlbums", "UpcomingSingers" })
+                {
+                    try
+                    {
+                        await connection.CreateDdlCommand($"DROP TABLE {table}")
+                            .ExecuteNonQueryAsync();
+                    }
+                    catch (SpannerException e) when (e.ErrorCode == ErrorCode.NotFound)
+                    {
+                        // Table does not exist.  Not a problem.
+                    }
+                }
+                return 0;
+            }
```

Comment:
the foreach seems a little bit overkill :)
```suggestion
            using (var connection = new SpannerConnection(connectionString))
            {
                DeleteTable("UpcomingAlbums");
                DeleteTable("UpcomingSingers");
                return 0;
            }
            void DeleteTable(string table)
            {
                try
                    {
                        await connection.CreateDdlCommand($"DROP TABLE {table}")
                            .ExecuteNonQueryAsync();
                    }
                    catch (SpannerException e) when (e.ErrorCode == ErrorCode.NotFound)
                    {
                        // Table does not exist.  Not a problem.
                    }
            }
```